docs/extend/modular-resources/configure.md

```diff
@@ -1,30 +1,171 @@
-2. [Add a **module**](#configure-your-module) referencing this executable to the configuration of your robot.
-3. [Add a new component or service](#configure-your-modular-resource) referencing the custom resource provided by the configured **module** to the configuration of your robot.
+{{% /alert %}}
 
-{{% alert title="Modules vs. modular resources" color="tip" %}}
+## Add a module from the Viam Registry
 
-A configured *module* can make one or more *modular resources* available for configuration.
+The Viam Registry is a central repository of modules from both Viam and the robotics community that allows you to easily extend Viam's capabilities on your robot.
+
+To add a module from the Viam Registry to your robot:
+
+1. Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+1. Click on the **Components** subtab and click the **Create component** button.
+1. Enter the name of the module you would like to add to your robot.
+   To find the name of a module you're interested in, you can:
+
+   - Start typing to search for modules by name.
+     Modules available from the Viam Registry will be listed under the `From Registry` section of the search results.
+   - [Browse the Viam Registry](https://app.viam.com/modules) directly to search available modules.
+
+   {{<imgproc src="extend/modular-resources/configure/add-module-from-registry.png" resize="400x" declaredimensions=true alt="The add a component modal showing results for the intel realsense module ">}}
+
+1. When you have entered the name of the module that you would like to add to your robot, select the matching module in the search results and click the **Add module** button.
+
+When you add a module from the Viam Registry, the custom modular component it provides appears under the **Components** subtab like any other component.
+You can also [configure the module itself](#configure-a-module-from-the-viam-registry) from its configuration pane in the **Modules** subtab.
+
+{{<imgproc src="extend/modular-resources/configure/conf-component-from-module.png" resize="400x" declaredimensions=true alt="The components subtab of the config tab showing the camera component configuration pane for the realsense module">}}
+
+If the module requires that specific **Atrributes** be configured, click the **URL** link in the module's configuration pane to view the specific documentation for that module on the module's Github page.
+
+To delete a module added from the Viam Registry, click the trash can icon in the upper-right corner of the module configuration pane in the **Components** tab.
+
+### Configure a module from the Viam Registry
+
+Once you have added a module from the Viam Registry, you can view and configure the module from the **Modules** subtab:
+
+1. Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
```

Comment:
```suggestion
1. Navigate to the **Config** tab of your robot's page in the [Viam app](https://app.viam.com).
```
[opt nit] I've been doing it this way

Reply:
146 v 79, going with more common, at least for now.